packages/desktop-client/src/components/mobile/rules/RulesListItem.tsx

```diff
@@ -11,187 +13,176 @@ import { ActionExpression } from '@desktop-client/components/rules/ActionExpress
 import { ConditionExpression } from '@desktop-client/components/rules/ConditionExpression';
 import { groupActionsBySplitIndex } from '@desktop-client/util/ruleUtils';
 
-const ROW_HEIGHT = 60;
-
 type RulesListItemProps = {
-  rule: RuleEntity;
-  onPress: () => void;
-};
+  value: RuleEntity;
+} & Omit<GridListItemProps<RuleEntity>, 'value'>;
```

Comment:
Nit: We can probably use `WithRequired` here
```suggestion
type RulesListItemProps = WithRequired<GridListItemProps<RuleEntity>, 'value'>
```

Reply:
TIL: we have this interface!

@cursor can you please implement the suggested change